### PR TITLE
[DOPS-2648] migrate x-networking to jenkins libs

### PR DESCRIPTION
**Pull request by baotn166**
# Task
[DOPS-2648] migrate x-networking to jenkins libs

# Description
- Migrate x-networking to jenkins libs

# Author
Signed-off-by: Bao Tran [tran@soramitsu.co.jp](mailto:tran@soramitsu.co.jp)

[DOPS-2648]: https://soramitsu.atlassian.net/browse/DOPS-2648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,47 +1,15 @@
...
-            currentBuild.result = 'FAILURE'
-        } finally {
-            cleanWs()
-        }
-    }
-}
\ No newline at end of file
+new org.soramitsu.mainLibrary().call(
+  agentLabel: "android",
+  skipSonar: true,
```
Comment (wpi86) [inline Jenkinsfile:5]: why?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,47 +1,15 @@
...
+  skipDojo: true,
```
Comment (wpi86) [inline Jenkinsfile:6]: why?

File: Jenkinsfile [hunk 3]
```diff
@@ -1,47 +1,15 @@
...
+  agentImage: "android-build-box-jdk11:latest",
```
Comment (wpi86) [inline Jenkinsfile:7]: should we pin version?

**Review by wpi86 — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (baotn166) [reply Jenkinsfile:5]: we have another ticket to enable, this is just for migration

File: Jenkinsfile (hunk 2 above)
Comment (baotn166) [reply Jenkinsfile:6]: we have another ticket to enable, this is just for migration

**Review by wpi86 — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]

**Pull request merged**